src/marker.ts

```diff
@@ -1,3 +1,7 @@
-export function marker<T extends string | string[]>(key: T): T {
+export function marker<T extends string | string[]>(
+  key: T,
+  params?: unknown,
+  lang?: string,
```

Comment:
Why did you update the signature? do we support resolving params like lang?

Reply:
I don't know what to say. You saw my comment :sweat_smile: 
![image](https://github.com/jsverse/transloco-keys-manager/assets/22544289/488e5e25-a958-4cf7-98c0-d1f86c4a33a4)
![image](https://github.com/jsverse/transloco-keys-manager/assets/22544289/334ca510-b13e-48d7-90d4-47dc6bbaa3ea)

This is a way of specifying the scope.
I don't think this is a definitive solution.
But it does get people unstuck.

It comes from resolving the marker function in the same way as the Transloco service and pipe.

<details>
<summary>Code</summary>

```ts
  if (content.includes('@ngneat/transloco')) {
    extractors.push(serviceExtractor, pureFunctionExtractor);
  }

  if (content.includes('@ngneat/transloco-keys-manager')) {
    extractors.push(markerExtractor);
  }

  ...

  extractors
    .map((ex) => ex(ast))
    .flat()
    .forEach(({ key, lang }) => {
      // Here, we solve three functions in the same way.
      const [keyWithoutScope, scopeAlias] = resolveAliasAndKeyFromService(
        key,
        lang,
        scopes
      );
      addKey({
        scopeAlias,
        keyWithoutScope,
        ...baseParams,
      });
    });
```

</details>

Maybe there's a way to improve this. You need to support the scope in marker in any case. https://github.com/jsverse/transloco-keys-manager/issues/178 https://github.com/jsverse/transloco-keys-manager/issues/132 https://github.com/jsverse/transloco-keys-manager/pull/179
But I grant you it has nothing to do with this PR. I can remove it if you like.

I'd like to finish the Angular subject quickly. :sweat_smile:

Reply:
I've just reverted to this change.
I propose to move forward and I was preparing another PR for that.
But after the release of a version.

Reply:
@kekel87 Missed the ref to the other issue.
I think it should be in another PR as you intended 👍